account_invoice_mass_sending/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Send invoices in mass mode.
```

Comment:
@RobinetDenisAcsone This module adds a mass sending feature on invoices, using asynchronous jobs.